pom.xml

```diff
@@ -513,7 +515,7 @@
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <configuration>
                     <sourcepath>
-                        ${project.build.directory}/generated-sources/delombok;
+                        ${delombok.output.dir};
```

Comment:
This looks like revert the previous change, which could lead IDE reports errors again. Have you tested it?